esmf_regrid/experimental/io.py

```diff
@@ -52,11 +52,18 @@ def save_regridder(rg, filename):
     regridder_type = rg.__class__.__name__
     if regridder_type == "GridToMeshESMFRegridder":
         src_grid = (rg.grid_y, rg.grid_x)
-        src_shape = [len(coord.points) for coord in src_grid]
+        if len(src_grid[0].shape) == 1:
```

Comment:
`src_grid[0].ndim` is equivalent to `len(src_grid[0].shape)` and easier to read I think. This can be changed repeatedly through this function